### ISSUE TITLE
Investigate optimal value of tau0 for NHC thermostat

### ISSUE THREAD

**Issue by danielhollas** [PHOTOX/ABIN#129]
The current value of the `tau0` is 0.001 ps, which is very very low, likely tailored towards the PIMD application (but even there it might be suboptimal). I think for most other MD programs such as Amber or Gromacs the value is more like 1 ps. But it is true they are tailored more towards simulations on nanoseconds (and beyond) timescales, whereas in out ab initio world we're typically reaching tens of picosecond. Shorter value guarantees that the system is equilibrated quickly towards target tempertature, but it might hinder sampling of slow modes / process such as difussion / barrier crossing. We should think about a good test case to verify this and choose a perhaps more reasonable value as the default, at least for classical MD. The other option would be to have no default and leave the choice to the user, but we should still do some testing and provides guidance.

**Comment by stepan-srsen** [comment]
Hi, I know this is an old topic, but I came across it again today. I saw the default value of 0.001 ps and I thought it was an error in the documentation. You cannot get reasonable results with this default if you're interested in ACFs or dynamical properties in general. The usual default is more like 0.1-1.0 ps, or you can define it as a multiple of the timestep.

EDIT: At least the predefined value in the CMD sample input should be different.

**Comment by danielhollas** [comment]
Yep, completely agree. We just need to be careful to only change the default for classical MD, the default for PIMD probably needs to be different.

@stepan-srsen if you have a good system to test this that would be great.

We should do this change for the upcoming v2.0 version (together with making global thermostat a default), I am tracking all the breaking changes in this milestone. https://github.com/PHOTOX/ABIN/milestone/5

**Comment by stepan-srsen** [comment]
Honestly, I don't think you can find a single optimal value. It depends on the modes of the system and the particular application. I would just set it to something reasonable, as other MD codes do (e.g. 0.1-1.0 ps or 100-500 times the timestep). If you want to test it on something, you can calculate infrared or Raman spectra of some anharmonic molecule from the corresponding autocorrelation function with different NHC time constants.
